.gitignore

```diff
@@ -297,5 +297,5 @@ package/
 src/obj/
 *.log
 IDEFindNavigatorScopes.plist
-.swiftpm/xcode/xcuserdata/travisprescott.xcuserdatad
-AutorestSwift.xcworkspace/xcuserdata/travisprescott.xcuserdatad
+.swiftpm/xcode/xcuserdata/
+AutorestSwift.xcworkspace/xcuserdata/
```

Comment:
This is to fix issues we've had with certain files sneaking into our commits.

Reply:
Thanks for fixing this.